vocabsieve/main.py

```diff
@@ -805,6 +805,9 @@ def setImage(self, content: Optional[QPixmap]) -> None:
         )
         self.image_viewer.setPixmap(content)
 
+    def getConvertToUppercase(self) -> bool:
+        return self.settings.value("convert_uppercase", False, type=bool)
```

Comment:
You used the wrong name here. It has to be the same as the name you registered in config, i.e. `capitalize_first_letter`